ComputerInterface/Views/GameSettings/CreditsView.cs

```diff
@@ -0,0 +1,48 @@
+﻿using ComputerInterface.ViewLib;
+using System.Text;
+using UnityEngine;
+
+namespace ComputerInterface.Views.GameSettings
+{
+    public class CreditsView : ComputerView
+    {
+        private readonly UISelectionHandler _selectionHandler;
```

Comment:
Why is there a selector handler? There isn't anything to select

Reply:
forgot to remove that lol